test/manual_test.py

```diff
@@ -127,12 +127,12 @@ def cleanup_server():
         # what happens in TaskManager. But this lets
         # us pass some basic things.
 
-        browser_params = load_default_params()[1][0]
+        browser_params = BrowserParams()
         if browser_params_file is not None:
             with open(browser_params_file, "r") as f:
-                browser_params.update(json.loads(f.read()))
+                browser_params.from_json(f.read())
         js_request = browser_params.js_instrument_settings
-        js_request_as_string = jsi.convert_browser_params_to_js_string(js_request)
+        js_request_as_string = json.dumps(js_request)
```

Comment:
Please revert the change in this line. The js_request isn't a valid json unfortunately, so we need to call out to that method.

Reply:
Sorry about that--but I don't see the function `convert_browser_params_to_js_string`, and that was throwing an error at me. Is the correct function `clean_js_instrumentation_settings`?

Reply:
Yes, that would be it. Sorry for letting this file get so out of date.

Reply:
Not a problem at all. Just committed with new name.